Automattic/jetpack
# My Jetpack: Move cards logic to Connected Product Card

This one moves the logic related to the cards from Product Card (a visual component) to the Connected Product Card, avoiding inserting business logic into the visual component.

## Proposed changes:

* Move `menuIsActive` logic from `ProductCard`.
* Introduce new flags in `ProductCard` to control menu default options.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Start `My Jetpack`.
* Play with `VideoPress`, `Social`, `Search` and `Backup` cards.
* They should provide the menu options if the card is not absent and has the option to show (product is active, and standalone is not installed or active ).
